CONTRIBUTING.md

```diff
@@ -92,7 +92,7 @@ A JSON with the following structure:
 
 ```json
 {
-    "image": "saturncloud/saturn:2020.07.08.1",
+    "image": "saturncloud/saturn:2021.01.13",
```

Comment:
obviously, we don't need to update the, but I like to keep dates relatively recent when it's easy